src/main/java/pageObject/SeznamSearchResultPage.java

```diff
@@ -18,7 +17,7 @@ public boolean isOpen() {
     return element().isDisplayed(firstSearchResult);
   }
 
-  public void openFirstSearchResult() throws PageIsNotOpenException {
+  public void openFirstSearchResult() {
     Asserts.assertIsOpen(this);
```

Comment:
Assert na zacatku metody nebyva. Opet, assert bych delal jen na konci stepu nebo testu, kde potrebuju vyhodnotit dobehnuti testu/stepu a jako vyhodnoceni mi to staci.